app/models/git/exercise.rb

```diff
@@ -129,9 +139,7 @@ def tooling_files
     # This includes meta files
     memoize
     def tooling_filepaths
-      filepaths.select do |filepath| # rubocop:disable Style/InverseMethods
-        !filepath.match?(track.ignore_regexp)
-      end
+      filepaths
```

Comment:
There's no special processing yet, but I still felt it was nice to have the method.